source/developer/mobile-developer-setup.rst

```diff
@@ -117,10 +189,122 @@ Android (Device)
+
+Important: <username> refers to the username or organization in GitHub that forked the repository.
+
+3. Change the directory to ``mattermost-mobile``.
+ - ``cd mattermost-mobile``
+
+4. Run ``make pre-run`` in order to install all the dependencies.
+ - It is highly important that you run everything with the make commands and avoid using npm or yarn to install dependencies.
+ 
+Make Commands Explained
+------------------------------------
+
+We've included a bunch of make commands in order to control the development flow and to ensure that everything works as expected. Always try and use these make commands unless what you trying to do can't be accomplished by one of these commands.
+
+Every make command has to be run from a terminal in the project's root directory.
+
+Commands to Prepare the App
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These make commands are used to install dependencies, to configure necessary steps before running or building the app, and to clean everything.
+
+ - **make pre-run**: Downloads any project dependencies needed and will set up the app assets. This is the equivalent of running the .yarninstall, .podinstall and dist/assets make commands.
+ - **make .yarninstall**: Downloads JavaScript and react native dependencies. Once this command finishes executing it will create a `.yarninstall` empty file in the project's root directory to indicate that this command already ran. If yarn updates your local dependencies in the `node_modules` directory, it will automatically run `make post-install` for you.
+ - **make .podinstall**: Downloads cocoapods dependencies needed to build the iOS app. Once this command finishes executing it will create a `.podinstall` empty file in the project's root directory to indicate that this command already ran.
+ - **make post-install**: Normally this command will run automatically if yarn detects that the project updated some or all of its JavaScript dependencies. Once yarn finishes executing, this command will set up a few post installation steps for you automatically to ensure that everything runs correctly.
+ - **make clean**: Removes all the downloaded dependencies, clears the cache of those dependencies and deletes any builds that were created. It will NOT reset the repo, so your current changes will still be there.
+ - **make dist/assets**: Builds the assets to be used including images, localization files and overriding any necessary files (see Override Assets & White Labeling for details).
+ 
+Commands to Run the App
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+These make commands are used to run the app on a device or emulator in the case of Android, and on a simulator in the case of iOS. (see *Running the App on a Device* for details).
+
+ - **make start-packager**: Runs the react-native packager used to bundle the JavaScript code. This command will execute *pre-run* to ensure the app is prepared.
```

Comment:
I think we user code formatting for commands, may want to switch to code instead of italics